demo/blog_writer/main.py

```diff
@@ -62,30 +61,47 @@ def get_user_agent_from_config_file(
     parser = ArgumentParser()
     parser.add_argument("--config", type=str, default="agent_config.yaml")
     parser.add_argument("--transcript", type=str, default="transcript.txt")
+    parser.add_argument("--blueprint", type=str, help="Optional JSON blueprint file to use")
```

Comment:
By default, arguments in `add_argument` are required. You will need to add a default value such as `default=None` to make it optional I think.

Reply:
Nevermind, I think this is no longer the case.